salt/orchestrate/edx/templates/ansible_env_config.yml.j2

```diff
@@ -0,0 +1,533 @@
+### XQUEUE ENVIRONMENT ###
+
+xqueue_env_config:
+  XQUEUES:
+    {% for key, value in xqueue.queues.items() %}
+      {{ key }}: {{ value }}
+    {% endfor %}
+{% raw %}
+  'RABBIT_VHOST': '/prod'
+  'XQUEUE_WORKERS_PER_QUEUE': 2
+  'LOGGING_ENV' : 'prod'
+  'LOG_DIR' : '/logs'
+  'SYSLOG_SERVER' : "{{ SYSLOG_SERVER }}"
+  'RABBIT_HOST' : 'localhost'
```

Comment:
Ah, good catch. This should be changed to the consul service DNS since we won't be using haproxy.